toolchain/parser/parser.cpp

```diff
@@ -1100,6 +1150,15 @@ auto Parser::HandleFunctionSignatureFinishState() -> void {
       break;
     }
     case TokenKind::OpenCurlyBrace(): {
+      if (stack_context_ == ParseContext::Interface) {
+        CARBON_DIAGNOSTIC(
+            MethodImplNotAllowed, Error,
+            "Method implementations are not allowed in interfaces.");
+        emitter_->Emit(*position_, MethodImplNotAllowed);
+        HandleFunctionError(state, /* skip_past_likely_end */ true);
```

Comment:
```suggestion
        HandleFunctionError(state, /*skip_past_likely_end=*/true);
```

The `=` is important in these comments since it's something clang-format understands to change spacing on.